src/main/java/flaxbeard/immersivepetroleum/api/crafting/reservoir/ReservoirHandler.java

```diff
@@ -268,27 +264,19 @@ private static List<ColumnPos> keepOutline(Level world, List<ColumnPos> poly){
 				for(int x = -1;x <= 1;x++){
 					if(ReservoirHandler.getValueOf(world, pos.x + 1, pos.z) == -1){
 						ColumnPos p = new ColumnPos(pos.x + 1, pos.z);
-						if(!set.contains(p)){
-							set.add(p);
-						}
```

Comment:
It's a _set_ :p 
![image](https://user-images.githubusercontent.com/40431605/183068044-b87c3257-baac-49a1-9227-c642d1997ef2.png)
That's what IDEA tells me for set documentation: they can't have duplicate objects, so that check is useless, because the set _already does it for you_

Reply:
Resolved via discord